src/App.tsx

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string,
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.action);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.action);
+  }
+
+  action = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: `The last pressed key is [${event.key}]` });
```

Comment:
```suggestion
    this.setState({ pressedKey: event.key });
```

Reply:
actually it's redundant to save in state value that never changed (I mean string "The last pressed key is")